test/dummy/app/components/dummy/atom/image_and_content_component.sass

```diff
@@ -0,0 +1,45 @@
+.d-atom-image-and-content
+  &__row
+    align-items: center
+
+  &__image
+    border-radius: $grid-gutter-half
+    overflow: hidden
+
+  &__content
+    display: flex
+    flex-direction: column
+    align-items: flex-start
+    gap: map-get($spacers, bb) 0
+
+  &__title
+    margin-top: 0
+
+  +media-breakpoint-down(lg)
+    &__title
+      +font-size-display-m
+
+  +media-breakpoint-down(md)
+    &__title
+      +font-size-display-s
+
+  +media-breakpoint-down(xs)
+    &__content
+      padding-top: $grid-gutter-width
+
+  +media-breakpoint-up(sm)
+    &:not(&--image-right) &__content
+      padding-left: $grid-gutter-width
+
+    &--image-right &__row
+      flex-direction: row-reverse
+
+    &--image-right &__content
+      padding-right: $grid-gutter-width
+
+  +media-breakpoint-up(md)
+    &:not(&--image-right) &__content
+      padding-left: $grid-gutter-base * 4
+
+    &--image-right &__content
+      padding-right: $grid-gutter-base * 4
```

Comment:
media-breakpoint mixiny jsou v 5ce posunute, takhle to nesedi - mensi text by mel byt uz drive

Reply:
Nerekl bych, nevim co myslis "v 5ce", ale prijde mi ze ty velikosti vsude sedi.

Reply:
Myslel jsem v bootstrap 5

Reply:
Na 850px by mely byt texty pro "M", ale jsou tam "L"

![image](https://github.com/sinfin/folio/assets/1286942/e9fc60aa-7826-45a1-8655-bd01a99fa3eb)

Reply:
Mas pravdu, myslel jsem ze ten tablet je mene jak 768px.